<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>general_settings</class>
 <widget class="QDialog" name="general_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search By</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Save At</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditSaveAt"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonSaveAt">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Max Download Image</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>147</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxSearchBy"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBoxMaxDownloadImg">
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
